src/XliffTasks.Tests/XamlRuleDocumentTests.cs

```diff
@@ -160,33 +678,205 @@ public void BasicLoadAndTranslate()
   <BoolProperty Name=""MyBoolProperty"" Description=""III"" />
   <StringProperty Name=""MyStringProperty"">
     <StringProperty.Metadata>
-      <NameValuePair Name=""TypeDescriptorText"" Value=""NNN"" xliff:LocalizedProperties=""Value"">
-        <!-- Value: My type descriptor text comment -->
-      </NameValuePair>
       <NameValuePair Name=""SearchTerms"" Value=""JJJ"" TranslatableProp1=""LLL"" TranslatableProp2=""MMM"" NonTranslatableProp3=""same"" xliff:LocalizedProperties=""TranslatableProp1;TranslatableProp2"">
         <!-- Value: My search terms comment -->
       </NameValuePair>
+      <NameValuePair Name=""TypeDescriptorText"" Value=""NNN"" xliff:LocalizedProperties=""Value"">
+        <!-- Value: My type descriptor text comment -->
+      </NameValuePair>
     </StringProperty.Metadata>
   </StringProperty>
 </Rule>";
 
-            XamlRuleDocument document = new();
-            StringWriter writer = new();
-            document.Load(new StringReader(source));
+            RunXamlTranslationTest(source, translations, expectedTranslation, expectedXlf);
+        }
+
+        
+        /* the purpose of this test is to ensure that deeply nested translations translate as expected, as well as attributes declared as a nested element:
+          such as:
+          <A Name="MyName">
+            <A.Description>MyName Description</A.Description>
+            <B>
+              <C>
+                <D>
+                  <NameValuePair Name="SearchTerms" TranslatableProp1="this-is-translated" xliff:LocalizedProperties="TranslatableProp1">
+                    <NameValuePair.Value>My Search Terms</NameValuePair.Value> 
+                </D>
+                ....
+    
+        */
+        [Fact]
+        public void LoadAndTranslateWithDeeplyNestedTranslationAndElementAttributeSyntax()
+        {
+            string source =
+@"<Rule Name=""MyRule""
```

Comment:
`"""` style comments would make this test data slightly easier to read. Ditto elsewhere.